satellite/console/service.go

```diff
@@ -466,6 +467,17 @@ func (s *Service) Token(ctx context.Context, email, password string) (token stri
 
 	user, err := s.store.Users().GetByEmail(ctx, email)
 	if err != nil {
+		users, err := s.store.Users().GetAllByEmail(ctx, email)
```

Comment:
Why would there be multiple users with the same password?

Reply:
@egonelbre we don't have unique constraint in users table for email column

at the moment we are able to create multiple users with same email until one of them confirm their email address.

also, in theory, there could be users with same email and password, because we can't prevent it

Reply:
We should add the constraint asap then.

Reply:
@stefanbenten 
imagine situation when someone will use ( accidentally or not ) your email to create own account.
To prevent it we should send confirmation email, to be sure that its email really yours.
But if we will have unique constraint on email column we will have unactivated account with such email.
And real owner of this email won't be able to create own account with his email.